### PR TITLE
cleaned up code

### PR DESCRIPTION
**Pull request by svenJonis138**

### CODE REVIEW

**Comment by claraj** [comment]
@KelseyStiff can you review?

**Comment by KelseyStiff** [comment]
let me know when your ready for review @svenJonis138

File: convert_state.py [hunk 1]
```diff
@@ -1,60 +1,56 @@
...
 
-    print(data)
-    
-    # Create a dictionary of states -> abbreviations, and a dictionary of abbreviations -> states 
-    stateAbbr = data #dictionary of state abbreviations keys and state name values 
-    stateAbbr2 = {}# dictionary of state name keys and state abbreviation values
-    # state key is the key in the state dictionary. state value is the value in the dictionary 
-    for statekey, statevalue in stateAbbr.items() :
-        stateAbbr2[statevalue] =  statekey
+    state_abbr = read_data  # dictionary of state abbreviations keys and state name values
```
Comment (PickleEric) [inline convert_state.py]: try using a more specific variable name like state_key_to_name

File: convert_state.py [hunk 2]
```diff
@@ -1,60 +1,56 @@
...
+    state_abbr2 = {}  # dictionary of state name keys and state abbreviation values
```
Comment (PickleEric) [inline convert_state.py]: same with variable above ie. state_key_to_abbrev

Comment (KelseyStiff) [reply convert_state.py]: This name could possibly be a little more descriptive so the comment wouldn't be necessary.

File: convert_state.py [hunk 3]
```diff
@@ -1,60 +1,55 @@
...
 
 
+def convert_abbreviation_to_state(dictionary):
+    """ converts abbreviation to state name """
+    user_input = input('Enter abbreviation name: ').upper()
+    result = dictionary.get(user_input)
+    if result is None:
+        print(' abbreviation not found')
+    else:
+        print('the state with the abbreviation   ' + user_input + ' is ' + result)
```
Comment (KelseyStiff) [inline convert_state.py]: some extra spaces here aren't needed.

**Review by KelseyStiff — COMMENTED** [review]
left a couple comments for possible little changes, overall looks great! 👍

File: convert_state.py (hunk 1 above)
Comment (svenJonis138) [reply convert_state.py]: good call!
done

File: convert_state.py (hunk 2 above)
Comment (svenJonis138) [reply convert_state.py]: good call!
done

File: convert_state.py (hunk 3 above)
Comment (svenJonis138) [reply convert_state.py]: got it

**Comment by svenJonis138** [comment]
@KelseyStiff I think this is all finished!

**Pull request closed without merging**